ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-master.yaml

```diff
@@ -166,6 +166,28 @@ tests:
     test:
     - ref: openshift-extended-test
     workflow: cucushift-installer-rehearse-aws-ipi-private
+- as: debug-winc-aws-ipi
+  optional: true
+  run_if_changed: test/extended/winc/
+  steps:
+    cluster_profile: aws-qe
+    dependency_overrides:
+      BUNDLE_IMAGE: quay.io/winc/wmco-bundle:latest
+      OO_INDEX: quay.io/winc/wmco-index:latest
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      FORCE_SUCCESS_EXIT: "no"
+      OO_CHANNEL: stable
+      OO_INSTALL_NAMESPACE: openshift-windows-machine-config-operator
+      OO_PACKAGE: windows-machine-config-operator
+      OO_TARGET_NAMESPACES: '!install'
+      SUB_INSTALL_NAMESPACE: openshift-windows-machine-config-operator
+      TEST_FILTERS: ~ChkUpgrade&;~ConnectedOnly&
+      TEST_SCENARIOS: Windows_Containers
+      TEST_TIMEOUT: "50"
+    test:
+    - ref: openshift-extended-test
```

Comment:
If you want to run this e2e, you don't need to add the above steps. For QE automation, as you know, your operator bundles will be stored in the quay.io/openshift-qe-optional-operators/ocp4-index:latest image. So, we enabled this step: https://github.com/openshift/release/tree/master/ci-operator/step-registry/enable-qe-catalogsource for the OLM operators sub team. So, for QE, you only need to contribute your e2e tests in the https://github.com/openshift/openshift-tests-private, we already integrated it to the Prow CI.